variables.tf

```diff
@@ -121,6 +121,7 @@ variable "password" {
   description = "Password for the master DB user. Note that this may show up in logs, and it will be stored in the state file"
   type        = string
   default     = null
+  sensitive   = true
```

Comment:
This would also require updating the minimum supported terraform version to 0.14.x. Found this closed issue that is related: https://github.com/terraform-aws-modules/terraform-aws-rds/issues/372